client/settings.gradle

```diff
@@ -0,0 +1,11 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ *
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user manual at https://docs.gradle.org/6.7/userguide/multi_project_builds.html
+ */
+
+rootProject.name = 'openlineage-client'
```

Comment:
_Minor_: I think appending the language type to the client name is a nice way to communicate support? `openlineage-java`, `openlineage-python`, `openlineage-go`, etc.

Reply:
since each language has their own artifact repository, I think it is easier to look for the -client artifact in each repo. maven would only have the java client etc.